ArduPlane/GCS_MAVLink_Plane.h

```diff
@@ -54,7 +54,11 @@ class GCS_MAVLINK_Plane : public GCS_MAVLINK
 
     void handle_message(const mavlink_message_t &msg) override;
     bool handle_guided_request(AP_Mission::Mission_Command &cmd) override;
+#if AP_MAVLINK_CHANGEALT_MISS_CURR_3_ENABLED
     void handle_change_alt_request(Location &location) override;
+#else
+    void handle_change_alt_request(Location &location);
```

Comment:
Why not `#if` it out of the cpp too?

Reply:
The method is used elsewhere in Plane, so we sitll need the method to be around.  It becomes non-virtual, 'though.